install.ts

```diff
@@ -0,0 +1,185 @@
+import {
+  ErrorKind,
+  Process,
+  ProcessStatus,
+  args,
+  chmod,
+  copyFile,
+  env,
+  exit,
+  lstat,
+  platform,
+  symlink,
+  writeFile,
+  removeAll,
+  rename,
+  run,
+  makeTempDir
+} from "deno";
+
+import { mkdirp } from "https://deno.land/x/std/mkdirp/mkdirp.ts";
+import { join, win32 } from "https://deno.land/x/path/index.ts";
+
+const WIN32: boolean = platform.os === "win";
+
+const proc_env: { [key: string]: any } = env();
+
+function get_home(): string {
+  return WIN32 ? win32.resolve("C:", proc_env.HOMEPATH) : proc_env.HOME;
```

Comment:
Instead of hard-coding `"C:"`, you could use `HOMEDRIVE`.

```suggestion
  return WIN32 ? win32.resolve(proc_env.HOMEDRIVE, proc_env.HOMEPATH) : proc_env.HOME;
```